### PR TITLE
test(uat): log rotation with minimum log level setting

### PR DESCRIPTION
**Pull request by MikeDombo**
**Issue #, if available:**

**Description of changes:**

**Why is this change necessary:**

**How was this change tested:**

**Any additional information or context required to review the change:**

**Checklist:**
 - [ ] Updated the README if applicable
 - [ ] Updated or added new unit tests
 - [ ] Updated or added new integration tests
 - [ ] Updated or added new end-to-end tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: uat/codebuild/uat_linux_buildspec.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ phases:
       - mvn -U -ntp verify -DskipTests=true
       - mvn -U -ntp clean verify -f uat/pom.xml
       - java -Dggc.archive=/tmp/greengrass-nucleus-latest.zip
-        -Dtags='LogManager&!unstable' -Dggc.install.root=$CODEBUILD_SRC_DIR -Dggc.log.level=INFO -Daws.region=us-west-2
+        -Dtags='MyTest&LogManager&!unstable' -Dggc.install.root=$CODEBUILD_SRC_DIR -Dggc.log.level=INFO -Daws.region=us-west-2
```
Comment (jcosentino11) [inline uat/codebuild/uat_linux_buildspec.yaml:17]: Looks leftover from testing

Comment (jcosentino11) [reply uat/codebuild/uat_linux_buildspec.yaml:17]: Ah didn't see was a draft PR :p

File: src/main/java/com/aws/greengrass/logmanager/services/DiskSpaceManagementService.java [hunk 2]
```diff
@@ -31,7 +31,13 @@ public List<String> freeDiskSpace(LogFileGroup group) {
 
         long bytesDeleted = 0;
         long minimumBytesToBeDeleted = Math.max(group.totalSizeInBytes() - group.getMaxBytes().get(), 0);
+        // Prefer to delete files we've already processed, however we do also need to clean up files that we
+        // have not processed as they may exceed the max disk space setting.
         List<LogFile> deletableFiles = group.getProcessedLogFiles();
+        deletableFiles.addAll(group.getLogFiles());
```
Comment (jcosentino11) [inline src/main/java/com/aws/greengrass/logmanager/services/DiskSpaceManagementService.java:37]: lazy question, `group.getLogFiles` distinct from `group.getProcessedLogFiles`?   seems like we dont want duplicates in delteableFiles

Comment (MikeDombo) [reply src/main/java/com/aws/greengrass/logmanager/services/DiskSpaceManagementService.java:37]: yes, they're distinct. Even if we duplicate it won't matter as deletion is safe if the file is deleted already.

**Pull request closed without merging**